### PR TITLE
[MOD-53][Feature] Add migration for reviews subscriptions

### PR DESCRIPTION
**Pull request by laurenbarker**
## Purpose

Add migration for reviews subscription.

## Ticket

https://openscience.atlassian.net/browse/MOD-53

### CODE REVIEW

File: osf/migrations/0061_auto_20171002_1438.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        for user in paginated_users.page(page_num).object_list:
+            user_subscription_id = to_subscription_key(user._id, 'global_reviews')
+
+            subscription = NotificationSubscription.load(user_subscription_id)
+            if not subscription:
+                logger.info('No {} subscription found for user {}. Subscribing...'.format('global_reviews', user._id))
+                subscription = NotificationSubscription(_id=user_subscription_id, owner=user, event_name='global_reviews')
+                subscription.save()  # Need to save in order to access m2m fields
+                subscription.add_user_to_subscription(user, 'email_transactional')
+                subscription.save()
```
Comment (aaxelb) [inline osf/migrations/0061_auto_20171002_1438.py]: `add_user_to_subscription` already saves the subscription

File: osf/migrations/0061_auto_20171002_1438.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+logging.basicConfig(level=logging.INFO)
+
+
+def add_reviews_notification_setting(*args, **kwargs):
+    active_users = OSFUser.objects.filter(date_confirmed__isnull=False).exclude(date_disabled__isnull=False).exclude(is_active=False).order_by('id')
+    total_active_users = active_users.count()
+
+    logger.info('About to add a global_reviews setting for {} users.'.format(total_active_users))
+
+    paginated_users = Paginator(active_users, 1000)
```
Comment (aaxelb) [inline osf/migrations/0061_auto_20171002_1438.py]: Is using `Paginator` better than `active_users.iterator()`?

Comment (laurenbarker) [reply osf/migrations/0061_auto_20171002_1438.py]: No idea, I was just copying another migration that used Paginator.

Comment (aaxelb) [reply osf/migrations/0061_auto_20171002_1438.py]: If we could bulk-create `NotificationSubscription` and the many-to-many, one batch per page, then `Paginator` would make sense. I think we should either do that or, if it's too messy, use `.iterator()`

**Comment by laurenbarker** [comment]
@aaxelb, updated 💰

**Comment by laurenbarker** [comment]
@aaxelb, rebased

**Review by aaxelb — APPROVED** [review]

**Pull request merged**